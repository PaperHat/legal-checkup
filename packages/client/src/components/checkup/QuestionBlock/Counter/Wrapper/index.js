import styled from 'styled-components'
import { DESKTOP, MOBILE, TABLET } from '../../../../../constants'
import styledMap from 'styled-map'

const Wrapper = styled.div`
  color: #b62645;
  width: 36px;
  height: 17px;
  font-family: MuseoSansCyrl;
  font-size: 13px;
  font-weight: 900;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  padding-top: 18px;
  ${styledMap('format', {
    [DESKTOP]: `
   diplay:none;
    `,
    [MOBILE]: `
    
   `,
    [TABLET]: `
  display:none;
    `
  })}
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
