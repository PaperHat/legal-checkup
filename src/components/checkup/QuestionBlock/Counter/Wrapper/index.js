import styled from 'styled-components'

const Wrapper = styled.div`
  color: #b62645;
  width: 50px;
  height: 17px;
  font-family: 'Museo Sans Cyrl 900', sans-serif;
  font-size: 13px;
  font-weight: 900;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  margin: 0px auto;
  padding-top: 18px;
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
