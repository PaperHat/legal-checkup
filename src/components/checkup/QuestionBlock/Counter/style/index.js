import styled from 'styled-components'

const Counter = styled.p`
  color: #b62645;
  font-family: 'Museo Sans Cyrl 500', sans-serif;
  font-size: 13px;
  font-weight: 900;
  text-align: center;
  margin: 0px auto;
  padding-top: 18px;
`
Counter.displayName = 'StyledCounter'

export default Counter
