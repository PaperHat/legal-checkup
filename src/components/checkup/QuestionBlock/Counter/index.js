import { connect } from 'react-redux'

import {
  getActiveQuestionIndex,
  getQuestionCount
} from '../../../../state/selectors'

import Wrapper from './Wrapper'

function mapStateToProps ({ checkup }) {
  const children = `${getActiveQuestionIndex(checkup) + 1} / ${getQuestionCount(
    checkup
  )}`

  return { children }
}

export default connect(mapStateToProps)(Wrapper)
