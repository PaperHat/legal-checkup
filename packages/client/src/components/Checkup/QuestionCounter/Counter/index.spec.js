import React from 'react'

import { shallow } from 'enzyme'

import Counter from './'

describe('components:styled:Counter', () => {
  it('renders the Counter and CSS properties properly', () => {
    expect(toJson(shallow(<Counter>Text</Counter>))).toMatchSnapshot()
  })
})
