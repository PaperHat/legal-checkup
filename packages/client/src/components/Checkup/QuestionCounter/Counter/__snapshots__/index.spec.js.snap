// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:Counter renders the Counter and CSS properties properly 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  width: 63px;
  height: 17px;
  font-family: Helvetica;
  font-size: 14px;
  font-weight: 300;
  font-style: oblique;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  text-align: center;
  color: #000000;
  margin: 0px auto;
",
        ],
      },
      "displayName": "Counter",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Text
</StyledComponent>
`;
